### ISSUE TITLE
CIS charts are not available even after adding the catalog/repo

### ISSUE THREAD

**Issue by sowmyav27** [rancher/rancher#28571]
**What kind of request is this (question/bug/enhancement/feature request):** bug

**Steps to reproduce (least amount of steps as possible):**
- Navigate to Apps --> Repositories -->Add a repo in dashboard URL: https://github.com/prachidamle/charts.git and branch: `cis-v2-chart`
- Go to charts
- CIS benchmark charts are NOT available.

**Expected Result:**
CIS benchmark charts should be available

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): master-head - commit id: `9b6984d5f`
- Installation option (single install/HA): HA

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported): imported rks k8s
- Kubernetes version (use `kubectl version`):

```
1.18
```

**Comment by prachidamle** [comment]
CIS chart wont be available until it is merged into rancher/charts
After merging some parts are automated to add it to the http://charts.rancher.io/index.yaml - The github actions flow for the rancher/charts repo does that once the chart is merged.

Until then a chart can be tested using usual `helm cli` by forking the chart branch to local setup

**Comment by sowmyav27** [comment]
Able to deploy charts using the helm command by cloning the chart branch into local.
